scripts/build-k8s-on-amd64/Dockerfile.arm64

```diff
@@ -24,7 +24,9 @@ ENV GOARM=6 \
     FLANNEL_VERSION=v0.5.5 \
     K8S_VERSION=v1.2.0 \
     REGISTRY_VERSION=v2.3.1 \
-    KUBE_BUILD_PLATFORMS="linux/arm64"
+    KUBE_BUILD_PLATFORMS="linux/arm64" \
+    TAG=0.61 \
```

Comment:
Oh, shame on me for not remembering that :smile:!
Then I would like you to rename to `INGRESS_CONTROLLER_VERSION`, and specify `TAG=1.0 ARCH=arm(64)` to `make server` when building `exechealthz`

If there are other places that require TAG too, specify them on the same line rather than a global env variable.